requirements/base.in

```diff
@@ -3,6 +3,7 @@
 
 fs
 lxml
+mako
```

Comment:
I get why, but it does pain me just a little that we're adding this.

Reply:
We have marked the function deprecated using it
https://github.com/openedx/XBlock/pull/669/files#diff-d87a75b05a573cc57741806b7884e3971a94dbbbfce99698485d16701717f469R53